pkg/plugin/action/backup/pod_action.go

```diff
@@ -22,32 +22,33 @@ import (
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/heptio/ark/pkg/apis/ark/v1"
+	v1 "github.com/heptio/ark/pkg/apis/ark/v1"
```

Comment:
Yes, I'm glad you noticed this, I agree with you. I looked thru the code and there's no reason for it to live under the plugin tree structure. 

I am moving it back to under `pkg/backup` but with one mod: I'm adding those files to a package named `backupaction`. It could alternatively be named `actionbackup`, or something else. Likewise for restore. I think separating those files keeps similar code together and could be beneficial when thinking about it. PTAL after I commit.